lib/telegraf/telegraf.conf.erb

```diff
@@ -562,7 +562,7 @@
 #   servers = ["127.0.0.1:27017"]
 
 
-<% config_file['mysql'].each do |server_url| %>
+<% (config_file['mysql'] || []).each do |server_url| %>
```

Comment:
instead of doing this here, why not add this as a variable to be bound in [`server_controller.rb`](https://github.com/Instrumental/instrumentald/blob/master/lib/instrumentald/server_controller.rb), so there you can set a default and just refer to the servers as `mysql_servers` or something?